apps/total-typescript/src/components/team/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import SelfRedeemButton from './self-redeem-button'
+import CopyInviteLink from './copy-invite-link'
+import Link from 'next/link'
+
+type InviteTeamProps = {
+  purchase: {
+    merchantChargeId: string | null
+    bulkCoupon: {id: string; maxUses: number; usedCount: number} | null
+    product: {id: string; name: string}
+  }
+  existingPurchase: {
+    id: string
+    product: {id: string; name: string}
+  }
+  session: any
+  setPersonalPurchase: (props: any) => void
+}
```

Comment:
*not this PR*: we talked about this in another PR, but it would be nice to consolidate this specific set of types which appear in a bunch of places across apps into either the SDK or commerce-server

Reply:
absolutely